torchrl/objectives/value/functional.py

```diff
@@ -351,7 +345,6 @@ def td1_return_estimate(
     return returns
 
 
-@_transpose_time
```

Comment:
There is no need to wrap the functions which call transpose_time functions.
We just pass the time_dim directly

Reply:
Yep but we need it in the docstrings and hence in the arguments!

Reply:
What do you mean, it is an argument and also in the docstring.

I just removed it as an argument from the td0 functions, as those do not need time

Reply:
Maybe not the advantages but I would keep it for td1 as in the future we might optimize this function to reduce the unnecessary overhead.

Reply:
I am not understanding. All the functions apart from td0 ones still have the argument and the docstring.

Reply:
Sorry I misread :)